code/lib/core-server/src/utils/get-story-id.ts

```diff
@@ -0,0 +1,44 @@
+import type { Options } from '@storybook/types';
```

Comment:
Whoa. What `Options` are these? That seems like a way too generic name for a top-level export on a generic `types` package.

Reply:
Hehe, that's the type for the monumental `options` object, which get's passed into addons/presets/... . It contains CLIOptions, Build/Dev options, Presets and a lot more.